dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
+
+namespace Microsoft.ProjectReunion
+{
+    // Enums duplicated from IPowerManagerStatics
+    enum EnergySaverStatus
```

Comment:
Is this the same as Battery Saver or something else?

Reply:
No, this is EnergySaverStatus as mentioned in https://docs.microsoft.com/en-us/uwp/api/Windows.System.Power.PowerManager?view=winrt-19041